content/en/containers/troubleshooting/_index.md

```diff
@@ -1,9 +1,210 @@
-  {{< nextlink href="/containers/troubleshooting/duplicate_hosts">}}<u>Duplicate hosts</u>: If you are running the Datadog Agent in a Kubernetes environment on AWS (fully self-managed on EC2, or using EKS) you may see a problem with duplicate hosts.{{< /nextlink >}}
-{{< /whatsnext >}}
\ No newline at end of file
+When deploying the Agent in a containerized environment, Datadog recommends following the documentation provided on this site. There are three methods of deploying the Agent:
+
+ 
+
+[As a container in a runtime](https://docs.datadoghq.com/containers/docker/?tab=standard) 
+
+In a Cloud environment => Common cloud environments are [Amazon ECS](https://docs.datadoghq.com/containers/amazon_ecs/?tab=awscli) [Fargate in an Amazon ECS environment](https://docs.datadoghq.com/integrations/ecs_fargate/?tab=webui#), and [Amazon EKS](https://docs.datadoghq.com/integrations/eks_fargate/#pagetitle).
+
+[In a Kubernetes environment](https://docs.datadoghq.com/containers/kubernetes/)
+
+ 
+
+As these processes may present unique deployment challenges, investigate this guide and use it as a starting point to resolve issues. If you continue to have trouble, reach out to [Datadog support](https://docs.datadoghq.com/help/) for further assistance. For details on Agent release updates or changes, please refer to Datadog’s Releases page (not specific to the Containers product area). 
+
+ 
+
+## General Issues: 
+
+ 
+
+**[Environment Variables](https://docs.datadoghq.com/agent/guide/environment-variables/#overview) are not being set and tags are not injected:**
+
+ A useful way to inject environment variables or to configure a DogStatsD library is to implement the Admissions Controller feature on the Cluster Agent. Please note that the Cluster Agent must be deployed and running BEFORE the application is deployed.
+
+ 
+
+**Metrics are not appearing on the Datadog Web Platform:**
+
+This can be due to many different factors, but it is important to ensure that the:
+
+ - Metrics endpoint is exposed and is open for the the Agent to reach.
+
+ - There are no proxies or firewalls that might impede the Agent from accessing the endpoint. 
+
+ - Agent has [Autodiscovery](https://docs.datadoghq.com/getting_started/containers/autodiscovery/?tab=adannotationsv2agent736#overview) enabled.
+
+ 
+
+**Not getting logs:**
+
+There are two [environment variables](https://docs.datadoghq.com/agent/guide/environment-variables/#overview) that can impact if logs will be collected and, if so, from which containers. The first variable is `DD_LOGS_ENABLED`, which must be enabled. To collect all logs from all containers, you can set the environment variable `DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL` to `true`.
+
+To exclude logs (and other features) from collection, please refer to Datadog’s [Container Discovery Management page](https://docs.datadoghq.com/agent/guide/autodiscovery-management/?tab=containerizedagent#pagetitle) for more insight.
```

Comment:
To exclude logs (and other features) from collection, see Datadog’s [Container Discovery Management page](https://docs.datadoghq.com/agent/guide/autodiscovery-management/?tab=containerizedagent#pagetitle) for more insight.